redditnav.js

```diff
@@ -71,26 +69,29 @@ function setUpButton($floatingButton, items) {
   $floatingButton.find('.mfb-component__button--main, .mfb-component__button--child').css('background-color', items.color);
   var buttonClass;
   switch(items.buttonPos){
-    case "right":
-      buttonClass = "mfb-component--br mfb-slidein-spring";
+    case 'right':
+      buttonClass = 'mfb-component--br mfb-slidein-spring';
       break;
-    case "left":
-      buttonClass = "mfb-component--bl mfb-slidein-spring";
+    case 'left':
+      buttonClass = 'mfb-component--bl mfb-slidein-spring';
       break;
-    case "hide":
-      buttonClass = "mfb-component--hide";
+    case 'hide':
+      buttonClass = 'mfb-component--hide';
       break;
   }
+
   $floatingButton.attr('class', buttonClass);
 }
 
 $(function() {
   chrome.storage.sync.get({
     color: '#FF5722',
     buttonPos: 'right'
-  }, function(items) {
-    $("head").append('<link href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css" rel="stylesheet" type="text/css">');
-    var $floatingButton = $('<ul class="mfb-component--br mfb-slidein-spring" data-mfb-toggle="hover">\
+  },
+
+  function(items) {
+    $('head').append('<link href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css" rel="stylesheet" type="text/css">');
+    var $floatingButton = $('<ul class="mfb-component--br mfb-slidein-spring" data-mfb-toggle="hover" style="z-index:999">\
```

Comment:
I have a feeling there will always be subreddit sidebars that push their z-indexes above any arbitrary constant we use. But that's a problem for the future; scouring the page and determining the highest z-index to compete with and one-upping would be non-performant and generally a bad idea.

👍